deploy/operator.yaml

```diff
@@ -41,6 +41,8 @@ spec:
           image: $SRIOV_NETWORK_OPERATOR_IMAGE
           command:
           - sriov-network-operator
+          args:
+          - --leader-elect=$OPERATOR_LEADER_ELECTION_ENABLE
```

Comment:
do we need this in the bundle also?

Reply:
Openshift bundle already has `--leader-elect` option set by default

https://github.com/openshift/sriov-network-operator/blob/master/manifests/stable/sriov-network-operator.clusterserviceversion.yaml#L366

Reply:
great thanks!

btw I am almost sure I already asked that :P